packages/reaction-i18n/private/data/i18n/ru.json

```diff
@@ -203,7 +203,9 @@
         "toggleSize": "Переключить размер",
         "deleteProduct": "Удалить товар",
         "publish": "Опубликовать",
-        "moreOptions": "Больше опций"
+        "moreOptions": "Больше опций",
+        "noSelectedProducts": "Нет выбранных продуктов. Нажмите на ",
+        "orShiftCtrlCmd": " либо используйте shift|ctrl|cmd + клик мыши на продуктах, чтобы выбрань несколько. С шифтом вы можете выбирать ряд смежных продуктов."
```

Comment:
no we can't. I wast few hours today to discover this). It wraps html tags in text quotes.